canonical-web-and-design/canonical.com
# Add content to the overview tab of careers pages

## Done
- Add content to the following careers pages:
- [x] `/careers/admin`
- [x] `/careers/engineering`
- [x] `/careers/marketing`
- [x] `/careers/travel`
- [x] `/careers/ethics`
- [x] `/careers/hr`
- [x] `/careers/legal`
- [x] `/careers/lifestyle`
- [x] `/careers/progression`
- [x] `/careers/diversity`
- Fix drop-down navigation on mobile version for all careers pages

## QA
- Go to [/careers/design](http://0.0.0.0:8002/careers/design) and navigate to all pages from left navigation menu. Compare the overview tab for the pages mentioned above to the content from [google drive](https://drive.google.com/drive/u/1/folders/1eppuIK_ETt7zIyrqDE6VDtDkjYlPJD4T)
- Make sure the drop-down navigation menu on mobile is functional

## Issue

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1566
